src/applications/gqrx/mainwindow.cpp

```diff
@@ -2300,3 +2307,20 @@ void MainWindow::on_actionAddBookmark_triggered()
         ui->plotter->updateOverlay();
     }
 }
+
+// Hide/show the right-hand settings menu, to give a larger FFT viewing area
+void MainWindow::on_actionShowRightDock_toggled(bool state)
+{
+    if(state){
+        if(stateBeforeMinimize != Q_NULLPTR)
+            restoreState(stateBeforeMinimize);
+    }
+    else{
+        //Save what the dock area looks like, then hide all dock widgets
+        stateBeforeMinimize = saveState();
+        for(QDockWidget* dw: findChildren<QDockWidget *>()){
+            if(dockWidgetArea(dw) == Qt::RightDockWidgetArea)
```

Comment:
The placement of the dock widgets is not limited to the right side of the window; however, the implementation will only work for dock widgets placed on the right side of the main window.